ckb-auth-rs/Cargo.toml

```diff
@@ -9,14 +9,14 @@ edition = "2021"
 default = ["dynamic-library-memory-200"]
 ckb2023 = ["ckb-std/ckb2023"]
 
-enable-dynamic-library = ["lazy_static"]
+enable-dynamic-library = ["lazy_static", "ckb-std/dlopen-c", "ckb-std/ckb-types"]
 dynamic-library-memory-200 = ["enable-dynamic-library"]
 # enable these features when memory is not enough
 dynamic-library-memory-400 = ["enable-dynamic-library"]
 dynamic-library-memory-600 = ["enable-dynamic-library"]
 
 [dependencies]
-ckb-std = "0.14.3"
+ckb-std = { version = "0.15", default-features = false }
```

Comment:
The feature allocator is missing.

Reply:
we didn't use `default_alloc` fn in this crate, no need to enable this feature of ckb-std.